gsoc/proposal-EfficientDataStorage.md

```diff
@@ -0,0 +1,112 @@
+## Google Summer of Code - Application Template
+
+### Student
+
+**Name:** Chaitanya Prakash Bapat, Chaitanya
+
+**Affiliation:** Sardar Patel Institute of Technology (University of Mumbai), Computer Engineering, Bachelor of Engineering (BE)
+
+**Location:** Mumbai, India and India Time Zone UTC+05:30
+
+**GitHub handle:** [@ChaiBapchya](https://github.com/ChaiBapchya)
+
+**Slack handle:** @chaitanyabapat
+
+**Email:** chai.bapat@gmail.com
+
+**Other Contact:**  
+
+**Twitter** : [@ChaiBapchya](https://twitter.com/ChaiBapchya)
+
+**LinkedIn** : [/in/chaitanyaprakashbapat](https://www.linkedin.com/in/chaitanyaprakashbapat)
+
+### Project Details
+
+**Project Title:**  
+
+Efficient Data Storage using Geo-temporal Clustering and Diff-based Storage
+
+**Project Abstract/Summary:**  
+
+Clustering the web pages (data) based on Geographical location and time in order to facilitate Machine Learning analysis.
+
+**Describe the need your project fills:**  
+
+_Broader Goal_ :- Better data management and a step towards creating Environment Data Warehouse
```

Comment:
By Environment Data Warehouse, I mean
All the data that ever existed and will exist revolving around Environment should necessarily be pooled into this Warehouse.
Such a robust source of information will provide immense analytical potential for Improving the existing environmental condition.
Warehouse will not just contain geo-temporal data of various environmental websites, but would act as a LiveHealth Tracker of sorts for anyone wanting to get environmental information